week-2-practise/level-3/6-map.js

```diff
@@ -4,18 +4,27 @@
 //    - every resulting item run through the `capitalise` function
 
 function tidyUpString(str) {
-  return str
+  if (str.charAt(0) != String) {
+    return str.replace("/", "").trim();
+  }
```

Comment:
This doesn't look right. It says "If the first character of the string is not equal to the string" (which is always true if the string has more than one character) then replace the character.

Why did you try to rewrite this function? It was already provided for you and the instructions only say to write the `capitalise` function.